xdist/plugin.py

```diff
@@ -26,8 +26,8 @@ def pytest_addoption(parser):
         help="shortcut for '--dist=load --tx=NUM*popen', "
              "you can use 'auto' here for auto detection CPUs number on "
              "host system")
-    group.addoption('--max-slave-restart', action="store", default=None,
-                    help="maximum number of slaves that can be restarted "
+    group.addoption('--max-worker-restart', action="store", default=None,
```

Comment:
IIRC you can pass multiple values here to support aliases:

```python
group.addoption('--max-worker-restart', '--max-slave-restart', action="store", ...
```

Reply:
I want to add dest value like:

`    group.addoption('--max-worker-restart', '--max-slave-restart', action="store", default=None,
                    dest="maxworkerrestart",
                    help="maximum number of workers that can be restarted "
                         "when crashed (set to zero to disable this feature)\n"
                         "'--max-slave-restart' option is deprecated and will be removed in next versions")`

Reply:
Seems good, perhaps just changing "in next versions" to "in a future release". 👍